src/main/proto/com/kodypay/grpc/pay/v1/pay.proto

```diff
@@ -29,6 +29,7 @@ message PayRequest {
   string store_id = 1; // UUID of store
   string amount = 2; // amount in BigDecimal/2.dp (0.00)
   string terminal_id = 3; // send the payment to this terminal serial number
+  optional bool show_tips = 4; // Show tips on the terminal
```

Comment:
Very nitipick: bool default is false so here I think we can simply use the non optional value https://protobuf.dev/programming-guides/proto3/#default

Reply:
But if not marked as optional, won't the generated code for statically typed languages have this as a breaking change? They will require the parameter when creating the message?

Reply:
java/kotlin don't care much about this (since we use builder pattern everywhere and values that are not set simply have their default values). 

Optional is more useful for distinguishing between default value and value not present

Reply:
I wanted to follow the same approach we have for other fields. Since not every request needs to have this, I declared it optional